delta/data/task/speaker_cls_task.py

```diff
@@ -582,3 +582,334 @@ def make_example(inputs, labels, filenames, clip_ids, soft_labels):
     return data.map(make_example, num_parallel_calls=10).\
                 batch(batch_size, drop_remainder=False).\
                 prefetch(tf.contrib.data.AUTOTUNE)
+
+
+class KaldiDir:
```

Comment:
Reuse KaldiDir if possible?

KaldiDir and data pipeline are already decoupled.